version/release.go

```diff
@@ -75,3 +82,26 @@ func FullInfo() string {
 
 	return strings.Join(res, "\n")
 }
+
+// FullInfoJson returns version information in JSON format.
+func FullInfoJson() string {
+	res := map[string]string{
+		"ProjectName": ProjectName,
+		"Version":     Version,
+		"PMMVersion":  PMMVersion,
+		"Timestamp":   timestampFormatted(),
+		"FullCommit":  FullCommit,
+	}
+	if Branch != "" {
+		res["Branch"] = Branch
+	}
+
+	b := &bytes.Buffer{}
+	encoder := json.NewEncoder(b)
+
+	if err := encoder.Encode(res); err != nil {
```

Comment:
let's use `json.Marshal()`
and in case of any error `panic` or return the error as the second value.

Reply:
done